config.ru

```diff
@@ -11,18 +11,21 @@ require "logger"
 require "app"
 
 in_development = ENV['RACK_ENV'] == 'development'
+log_path = ENV.fetch("LOG_PATH", in_development ? nil : "log")
 
 if in_development
   set :logging, $DEBUG ? Logger::DEBUG : Logger::INFO
-else
+end
+
+if log_path
   enable :logging
-  log = File.new("log/production.log", "a")
+  log = File.new(log_path + "/#{ENV['RACK_ENV']}.log", "a")
   log.sync = true
   STDOUT.reopen(log)
   STDERR.reopen(log)
 
   use Rack::Logstasher::Logger,
-    Logger.new("log/production.json.log"),
+    Logger.new(log_path + "/#{ENV['RACK_ENV']}.json.log"),
```

Comment:
Sorry - I'd expect log_path to be full path to log file. See https://github.com/alphagov/publishing-e2e-tests/blob/master/docker-compose.yml#L58